backend/__init__.py

```diff
@@ -7,10 +7,20 @@
 
 from backend import constants
 from backend.authentication import JWTAuthenticationBackend
-from backend.route_manager import create_route_map
 from backend.middleware import DatabaseMiddleware, ProtectedDocsMiddleware
+from backend.route_manager import create_route_map
 from backend.validation import api
 
+ORIGINS = [
+    r"(https://[^.?#]*--pydis-forms\.netlify\.app)",  # Netlify Previews
+    r"(https?://[^.?#]*.forms-frontend.pages.dev)",  # Cloudflare Previews
+]
+if not constants.PRODUCTION:
```

Comment:
Linebreak for clarity.

```suggestion

if not constants.PRODUCTION:
```

Reply:
Addressed in ca73008.